creating-campaign/stripe-payment-feature/stripe.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Stripe Connected Payments'
+---
+
+<Note>To enable the Stripe feature and connect your account to Moongate's Stripe account, please contact Moongate for permission.</Note>
+
+## What is Stripe Connect?
+> Stripe Connect is a service that allows platforms to manage payments on behalf of their users. Here's an example to illustrate how it works:
+>
+> Imagine you are an event organizer using Moongate to host and manage your events. You want to sell tickets online and receive payments from attendees. Instead of setting up your own payment processing system, you can use Stripe Connect. Moongate, as the platform, handles the payment processing for you. This means:
+>
+> **Moongate:** Manages the transactions, ensuring payments from ticket sales go through smoothly. <br />
+>
+> **Organizer (You):** Receives payments from ticket sales directly to your account without having to worry about the complexities of payment processing.
+> This allows you to focus on organizing your event while Moongate and Stripe Connect take care of the financial transactions.
+
+<Steps>
+    <Step title="Contact Moongate and Receive a One-Time Link">
+        1. Contact Moongate by clicking <a href="mailto:support@moongate.id">**Contact Support**</a> and request a Stripe connected account from our customer support team. <br />
+        2. Moongate will then send you a **One-Time Link**, which will look like this: <br />
+           <img src="/images/creating-campaign/stripe-payment-feature/one-time-link.png" />
+    </Step>
+    <Step title="Complete the Form from the One-Time Link">
+        Use the provided **One-Time Link** to access the Stripe Connect form. Fill out all required information to enable the Stripe feature. <br />
+        <img src="/images/creating-campaign/stripe-payment-feature/stripe-connect-site.png" />
+    </Step>
+</Steps>
+
+<Check>Once you have completed all of this, you will be connected to Moongate's Stripe account.</Check>
+<Warning>After you have finished entering your details, please our customer support team again so we can verify everything for you.</Warning>
```

Comment:
Use chatgpt to correct grammatical mistakes, also please put this point above the `<Check>`